glib/src/any_gobject.rs

```diff
@@ -0,0 +1,108 @@
+use crate as glib;
+use crate::subclass::prelude::*;
+use crate::{object_subclass, wrapper, Object};
+use std::any::Any;
+use std::cell::{Ref, RefCell, RefMut};
+
+mod imp {
+    use super::*;
+
+    #[derive(Debug)]
+    pub struct AnyGObject {
+        pub item: RefCell<Option<Box<dyn Any>>>,
+    }
+    impl Default for AnyGObject {
+        fn default() -> Self {
+            Self {
+                item: RefCell::new(None),
+            }
+        }
+    }
+    #[object_subclass]
+    impl ObjectSubclass for AnyGObject {
+        const NAME: &'static str = "AnyGObject";
+        type Type = super::AnyGObject;
+        type ParentType = Object;
+    }
+    impl ObjectImpl for AnyGObject {}
+}
+
+wrapper! {
+    pub struct AnyGObject(ObjectSubclass<imp::AnyGObject>);
+}
+
+impl AnyGObject {
+    pub fn new(item: Box<dyn Any>) -> Self {
```

Comment:
Should maybe take an `Option`, also the other API?

Reply:
And maybe the constructor and replace should take a `T: Any + 'static` and all the boxing happens internally?

Reply:
> take an Option


Makes sense



> also the other API?


I don't understand, what's the other API?
I didn't make the `Send` one because I didn't know what to do. But now I know, so I'll do it.

Reply:
Ok, I guess the "other API" is replace. I've seen your second comment only after replying.

Reply:
Yes